modules/Administration/Search/MVC/View.php

```diff
@@ -77,8 +78,12 @@ public function preDisplay()
         if (!isset($sugar_config['search']) || !$sugar_config['search']) {
             LoggerManager::getLogger()->warn('Configuration does not contains default search settings.');
         }
-        $search = isset($sugar_config['search']) ? $sugar_config['search'] : null;
+        $search = $sugar_config['search'] ?? null;
         $this->smarty->assign('config', $search);
+
+        $cfg = new Configurator();
+        $this->smarty->assign('aod_config', $cfg->config['aod']);
```

Comment:
Hi @Dillon-Brown, where `aod_config` is used in the html/tpl?

Reply:
Unused, nice spot. Pushed!

Reply:
thanks